client.go

```diff
@@ -228,7 +228,7 @@ func startContainer(img string, cmd []string, streaming, wait bool) (string, err
 			}
 		}()
 		go func() {
-			if _, err := io.Copy(os.Stdout, rc); err != nil {
+			if _, err := io.Copy(dst, rc); err != nil {
```

Comment:
Good question. `dst` is only used if `streaming` is true. To be safe, I should do
```go
if streaming && dst != nil {
...
}
```